roles/create_cf_stack/templates/coreos-beta-swarm-hvm.template

```diff
@@ -574,7 +581,7 @@
           }
         ],
         "HealthCheck" : {
-          "Target" : { "Fn::Join" : [ "", ["HTTP:2379", "/health"]]},
+          "Target" : { "Fn::Join" : [ "", ["HTTP:4243", "/info"]]},
```

Comment:
i never thought of ETCD, assuming docker running on master node too, i changed it rely on docker port for health. Should i revert it back?